test/upgrade.test.js

```diff
@@ -1,53 +1,55 @@
 'use strict'
 
-const { test, skip } = require('tap')
+const { test } = require('node:test')
 const Fastify = require('..')
 const { connect } = require('node:net')
 const { once } = require('node:events')
 const dns = require('node:dns').promises
 
-async function setup () {
+test('upgrade to both servers', async t => {
   const localAddresses = await dns.lookup('localhost', { all: true })
   if (localAddresses.length === 1) {
-    skip('requires both IPv4 and IPv6')
+    t.skip('requires both IPv4 and IPv6')
     return
   }
 
-  test('upgrade to both servers', async t => {
+  await t.test('upgrade IPv4 and IPv6', async t => {
```

Comment:
use describe and test

Reply:
done, see commit here: https://github.com/fastify/fastify/pull/5917/commits/738fcd9c72b5e861dc5dcfb90ce7ea05e74c160f